lib/components/Atom/WheelPicker/index.tsx

```diff
@@ -0,0 +1,231 @@
+            dataRefs.current[dataItemsMap.get(selectedValue)]?.focus();
+        }
+    }, [focus]);
+
+    return (
+        <div
+            className={clsx(
+                "quill-wheel-picker__container",
+                containerClassName,
+            )}
+        >
+            <ul
+                className={clsx(
+                    "quill-wheel-picker__data-items",
+                    listClassName,
+                )}
+                role="listbox"
+                ref={dataItemsContRef}
+            >
+                {data.map(({ value }, index) => {
+                    return (
+                        <DropdownItem
+                            {...rest}
+                            className={clsx(
+                                "quill-wheel-picker__data-item",
+                                dropDownItemClassName,
+                            )}
+                            label={value}
+                            role="option"
+                            tabIndex={0}
+                            textAlignment="center"
+                            ref={(node: HTMLDivElement) =>
+                                (dataRefs.current[index] = node)
+                            }
+                            key={index}
+                            onKeyDown={handleKeyDown}
+                            onClick={() => {
+                                dataRefs.current[index]?.scrollIntoView({
+                                    block: "center",
+                                });
+                                dataRefs.current[index]?.focus();
+                                setSelectedValue(data[index].value);
+                            }}
+                            disabled={data[index].value !== selectedValue}
+                        >
+                            {value}
+                        </DropdownItem>
+                    );
+                })}
+            </ul>
+        </div>
+    );
+};
+
+export interface WheelPickerContainerProps extends InputProps {
+    data: { value: string | number }[][];
+    dataValues: (string | number)[];
+    setdataValues: (index: number, value: string | number) => void;
+    children?: React.ReactNode;
+    close?: () => void;
+    setSelectedValue?: (value: string) => void;
+}
+
+const WheelPickerContainer = ({
+    data = [],
+    dataValues = [],
+    setdataValues,
+    close,
+    setSelectedValue,
+    ...rest
+}: WheelPickerContainerProps) => {
+    const [colRef, setColRef] = useState<boolean[]>(
+        new Array(data.length).fill(false),
+    );
+
+    useEffect(() => {
+        if (setSelectedValue)
+            setSelectedValue(
+                `${dataValues[0] ?? ""} ${dataValues[1] ?? ""} ${dataValues[2] ?? ""}`,
+            );
+    }, [dataValues]);
+
+    const handleKeyDown = (e: KeyboardEvent<Element>, index: number) => {
+        setColRef(new Array(data.length).fill(false));
+        switch (e.key) {
+            case "ArrowRight":
```

Comment:
In _lib/utils/common-utils.ts_ there is an object for key names) You can reuse it if you want)